src/main/java/org/sagebionetworks/bridge/spring/controllers/UploadTableController.java

```diff
@@ -31,6 +43,50 @@ public final void setUploadTableService(UploadTableService uploadTableService) {
         this.uploadTableService = uploadTableService;
     }
 
+    /** Get the upload table job result, with the downloadable S3 URL if it's ready. */
+    @GetMapping("/v5/studies/{studyId}/uploadtable/requests/{jobGuid}")
+    public UploadTableJobResult getUploadTableJobResult(@PathVariable String studyId, @PathVariable String jobGuid) {
+        UserSession session = getAuthenticatedSession(STUDY_DESIGNER, STUDY_COORDINATOR, DEVELOPER, RESEARCHER);
+        return uploadTableService.getUploadTableJobResult(session.getAppId(), studyId, jobGuid);
+    }
+
+    /** List upload table jobs for the given app and study. Does not include the downloadable S3 URL. */
+    @GetMapping("/v5/studies/{studyId}/uploadtable/requests")
+    public PagedResourceList<UploadTableJob> listUploadTableJobsForStudy(@PathVariable String studyId,
+            @RequestParam(required = false) String start, @RequestParam(required = false) String pageSize) {
+        UserSession session = getAuthenticatedSession(STUDY_DESIGNER, STUDY_COORDINATOR, DEVELOPER, RESEARCHER);
+
+        int startInt = BridgeUtils.getIntOrDefault(start, 0);
+        int pageSizeInt = BridgeUtils.getIntOrDefault(pageSize, API_DEFAULT_PAGE_SIZE);
+
+        return uploadTableService.listUploadTableJobsForStudy(session.getAppId(), studyId, startInt, pageSizeInt);
+    }
+
+    /** Request a zip file with CSVs of all uploads in this app and study. */
+    @PostMapping("/v5/studies/{studyId}/uploadtable/requests")
```

Comment:
Not sure I understand why this is a post?

Reply:
Philosophically, get is "I'm asking the server for information" and post is "I'm asking the server to do something". For this method, you're asking the server to do something (start a worker job to aggregate the CSVs), so this would be a post. It's also functionally equivalent to a create (creating the worker job).